test/odbc/psqlodbc/numeric.cpp

```diff
@@ -0,0 +1,575 @@
+    COL1_NAME + "<=" + COL2_NAME,
+    COL1_NAME + ">" + COL2_NAME,
+    COL1_NAME + ">=" + COL2_NAME
+  };
+
+  // initialization of expected_results
+  vector<vector<char>> expected_results = {};
+
+  for (int i = 0; i < NUM_OF_DATA; i++) {
+    expected_results.push_back({});
+    const float data_A = std::stof(INSERTED_PK[i]);
+    const float data_B = std::stof(INSERTED_DATA[i]);
+    expected_results[i].push_back(data_A == data_B ? '1' : '0');
+    expected_results[i].push_back(data_A != data_B ? '1' : '0');
+    expected_results[i].push_back(data_A <  data_B ? '1' : '0');
+    expected_results[i].push_back(data_A <= data_B ? '1' : '0');
+    expected_results[i].push_back(data_A >  data_B ? '1' : '0');
+    expected_results[i].push_back(data_A >= data_B ? '1' : '0');
+  }
+
+  createTable(ServerType::MSSQL, BBF_TABLE_NAME, TABLE_COLUMNS);
+
+  insertValuesInTable(ServerType::MSSQL, BBF_TABLE_NAME, insertString, NUM_OF_DATA);
+
+  testComparisonOperators(ServerType::MSSQL, BBF_TABLE_NAME, COL1_NAME, COL2_NAME, 
+                          INSERTED_PK, INSERTED_DATA, BBF_OPERATIONS_QUERY, expected_results,
+                          false, false);
+
+  testComparisonOperators(ServerType::PSQL, PG_TABLE_NAME, COL1_NAME, COL2_NAME, 
+                          INSERTED_PK, INSERTED_DATA, PG_OPERATIONS_QUERY, expected_results,
+                          false, false);
+
+  dropObject(ServerType::PSQL, "TABLE", PG_TABLE_NAME);
+  dropObject(ServerType::MSSQL, "TABLE", BBF_TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Numeric, Arithmetic_Operators) {
+
+  const string COL1_DATATYPE = DATATYPE + "(1,0)";
+  const string COL2_DATATYPE = DATATYPE + "(5,5)";
+  const string COL1_NAME = "col1";
+  const string COL2_NAME = "col2";
+
+  vector<pair<string, string>> TABLE_COLUMNS = {
+    {COL1_NAME, COL1_DATATYPE + " PRIMARY KEY"},
+    {COL2_NAME, COL2_DATATYPE}
+  };
+
+  vector <string> inserted_pk = {
+    "8"
+  };
+
+  vector <string> inserted_data = {
+    "0.55555"
+  };
+
+  vector <string> pg_operations_query = {
+    COL1_NAME + "+" + COL2_NAME,
+    COL1_NAME + "-" + COL2_NAME,
+    COL1_NAME + "*" + COL2_NAME,
+    COL1_NAME + "/" + COL2_NAME,
+    "ABS(" + COL1_NAME + ")",
+    "POWER(" + COL1_NAME + "," + COL2_NAME + ")",
+    "||/ " + COL1_NAME,
+    "LOG(" + COL1_NAME + ")"    
+  };
+
+  vector<vector<string>>pg_expected_results;
+  vector<vector<string>>bbf_expected_results;
+
+  // initialization of expected_results
+  for (int i = 0; i < inserted_pk.size(); i++) {
+    pg_expected_results.push_back({});
+    pg_expected_results[i].push_back("8.55555");
+    pg_expected_results[i].push_back("7.44445");
+    pg_expected_results[i].push_back("4.44440");
+    pg_expected_results[i].push_back("14.4001440014400144");
+    pg_expected_results[i].push_back("8");
+    pg_expected_results[i].push_back("3.1747654273961317");
+    pg_expected_results[i].push_back("2");
+    pg_expected_results[i].push_back("0.9030899869919436");
+  }
```

Comment:
Maybe remove the for loops since only 1 pk and data for test, the iteration is one